lib/process-services/src/lib/form/widgets/dynamic-table/dynamic-table.widget.ts

```diff
@@ -179,8 +178,6 @@ export class DynamicTableWidgetComponent extends WidgetComponent implements OnIn
                 this.content.selectedRow.value = this.copyObject(this.editRow.value);
             }
             this.content.flushValue();
-        } else {
-            this.logService.error(this.ERROR_MODEL_NOT_FOUND);
```

Comment:
I think now `ERROR_MODEL_NOT_FOUND` can be removed as this was the only place it was used